CMakeLists.txt

```diff
@@ -6,9 +6,14 @@ else()
   cmake_policy(VERSION 3.12)
 endif()
 
+# CMake currently doesn't support proper semver
+# Set the version here, strip the prerelease identifier to use in `project`
+set(BOUT_FULL_VERSION 5.0.0-alpha)
+string(REPLACE "-alpha" "" BOUT_CMAKE_ACCEPTABLE_VERSION ${BOUT_FULL_VERSION})
```

Comment:
Is there a way to chop off everything after an `-` so we can have other labels if needed later?

Reply:
Good point! Looks like `REGEX REPLACE` will do it